specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -265,3 +265,781 @@ model TextProtectedMaterialAnalysisResult {
+model AddImageIncidentSamplesResult {
+  @doc("Array of incidentSamples have been added.")
+  incidentSamples: ListImageIncidentSampleResult[];
+}
+
+@added(ContentSafety.Versions.v2024_09_15_Preview)
+@doc("Definition of an image in a response.")
+model ImageDataResult {
+  @doc("Base64-encoded image.")
+  content?: bytes;
+}
+
+@added(ContentSafety.Versions.v2024_09_15_Preview)
+@doc("Definition of an Image Incident sample in a response.")
+@resource("incidentSamples")
+@parentResource(ImageIncident)
+model ImageIncidentSampleResult {
+  @doc("The name of this incident sample.")
+  @key("incidentSampleId")
+  @visibility("read")
+  incidentSampleId: string;
+
+  @doc("The image content of this incident sample.")
+  image?: ImageDataResult;
+}
+
+@added(ContentSafety.Versions.v2024_09_15_Preview)
+@doc("The request to remove incident samples from an Image Incident.")
+model RemoveImageIncidentSamplesOptions {
+  @doc("Incident samples to be removed.")
+  incidentSampleIds: string[];
+}
+
+@added(ContentSafety.Versions.v2024_09_15_Preview)
+@doc("The request to analyze given image with content safety Image Incidents defined.")
+model AnalyzeImageDetectIncidentsOptions {
+  @doc("The image to be analyzed.")
+  image: ImageData;
+
+  @doc("The incidents defined.")
+  incidentNames: string[];
+}
+
+@added(ContentSafety.Versions.v2024_09_15_Preview)
+@doc("The details of a long running operation of Text Incident.")
+@resource("text/incidents/operations")
+model TextIncidentOperationResult {
+  @doc("The id of this operation.")
+  @key("operationId")
+  @visibility("read", "create", "query")
+  id: string;
+
+  @doc("The status of this operation.")
+  @visibility("read")
+  status: IncidentOperationStatus;
+
+  @doc("The error details when this operation failed.")
+  errorMessage?: Azure.Core.Foundations.Error;
+}
+
+@added(ContentSafety.Versions.v2024_09_15_Preview)
+@doc("The details of a long running operation of Image Incident.")
+@resource("image/incidents/operations")
+model ImageIncidentOperationResult {
+  @doc("The id of this operation.")
+  @key("operationId")
+  @visibility("read", "create", "query")
+  id: string;
+
+  @doc("The status of this operation.")
+  @visibility("read")
+  status: IncidentOperationStatus;
+
+  @doc("The error details when this operation failed.")
+  error?: Azure.Core.Foundations.Error;
+}
+
+@added(ContentSafety.Versions.v2024_09_15_Preview)
+@doc("Incident long running operation parameters.")
+model IncidentLongRunningOperationParameters {
+  @doc("The operation id passed by the user, if not provided, will be automatically generated as a UUID.")
+  @header("Operation-Id")
+  @pattern("^(?=[a-zA-Z0-9])[a-zA-Z0-9-]*[a-zA-Z0-9]$")
+  @maxLength(64)
+  operationId?: string;
+}
+
+@added(ContentSafety.Versions.v2024_09_15_Preview)
+@doc("Incident long running operation status result.")
+model IncidentLongRunningOperationStatusResult {
+  @doc("The HTTP status code.")
+  statusCode: 202;
```

Comment:
Please double check.

Reply:
I believe this is from some outdated implementation. Just removed it.